app/views/layouts/card_wide.html.slim

```diff
@@ -1,3 +1,3 @@
-- card_cls 'card-wide'
+- card_cls 'card-wide sm-px2 px0'
```

Comment:
ah right, I had to manually add padding in `card-wide` and forgot to remove this. Thanks!